package.json

```diff
@@ -10,7 +10,7 @@
     "build:remix": "cross-env NODE_ENV=production remix build",
     "dev": "run-p dev:*",
     "dev:css": "npm run generate:css -- --watch",
-    "dev:remix": "cross-env NODE_ENV=development netlify dev",
+    "dev:remix": "remix dev",
```

Comment:
This allows you to run `npm run dev` and have `remix dev` run so that you can have hot module reloading while doing local development.